contributing_to_docs/doc_guidelines.adoc

```diff
@@ -59,6 +47,24 @@ If using section anchors, they must be all lowercase letters, with no line space
 === Section Title
 ----
 
+== Titles, headings, and filenames
+There are some general guidelines and conventions we follow for titles, headings, and topic filenames. These are described in the following table.
+
+[cols="4,5,8",options="header"]
+|===
+
+| |Example |Description
+
+.^|Topic filenames
+.^|manage_authorization_policies.adoc
+|Try to shorten the topic filename as much as possible WITHOUT abbreviating important terms that may cause confusion. In this example, this topic filename would be appropriate for a topic titled Managing Authorization Policies.
+.^|Topic / section titles
+.^|Managing Authorization Policies
+|Try to be as descriptive as possible with the topic title or section headings without making them unnecessarily too long. Most importantly, we use title case in all topic titles and section headings. See http://titlecase.com/ for more information.
+|===
```

Comment:
@CowboysFan Might not be a big deal, but we're asking the reader to use title case, but the section headings in this doc aren't using title case. Should they be? Or if we agreed not to use title case, should we reword this sentence?